contracts/non-fungible-tokens/andromeda-crowdfund/src/testing/tests.rs

```diff
@@ -41,6 +43,21 @@ fn get_purchase(token_id: impl Into<String>, purchaser: impl Into<String>) -> Pu
     }
 }
 
+fn generate_economics_message(payee: &str, action: &str) -> SubMsg {
```

Comment:
Is there somewhere we can put this to reduce redundancy? Maybe in `andromeda-testing`?